OutOfSchool/OutOfSchool.DataAccess/Models/PhoneNumber.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.ComponentModel.DataAnnotations;
+using OutOfSchool.Common;
+using OutOfSchool.Common.Validators;
+
+namespace OutOfSchool.Services.Models;
+
+public class PhoneNumber : IKeyedEntity<long>
+{
+    public long Id { get; set; }
+
+    [DataType(DataType.PhoneNumber)]
+    [Required(ErrorMessage = "Phone number is required")]
+    [CustomPhoneNumber(ErrorMessage = Constants.PhoneErrorMessage)]
+    [DisplayFormat(DataFormatString = Constants.PhoneNumberFormat)]
+    [MaxLength(Constants.MaxPhoneNumberLengthWithPlusSign)]
+    public string Number { get; set; }
+
+    public string Type { get; set; }
```

Comment:
Just curios: this `string Type` property - what it describes? Why `string` (not `enum`), why it is presetn for `Email` and `PhoneNumber`, but absent for `Website`

Reply:
The type property is something like a user note for Email, PhoneNumber, or Website (I missed adding it, but it's fixed now). For example, an entity can have many phone numbers associated with it, such as the owner's number, manager's number, etc.

Reply:
(Don't insist, just recommend) Then probably name it like `UserNotes` or `UserDescription`?